test/integration/game_navbar_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.